src/App.jsx

```diff
@@ -9,36 +11,17 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">
-              Tab 1
-            </a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">
-              Tab 2
-            </a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">
-              Tab 3
-            </a>
-          </li>
-        </ul>
-      </div>
+export const App = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```

Comment:
You use only `selectedTab.id` so probbaly it makes sense to store only tab id here

```suggestion
  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
```

Reply:
Thank you for this! However, in the default it was stated that by default, 1 table could be selected. Made the rest of the changes